docs/execution-providers/CUDA-ExecutionProvider.md

```diff
@@ -338,8 +338,11 @@ shown below) if [N, C, 1, D] is preferred.
 
 While using the CUDA EP, ORT supports the usage
 of [CUDA Graphs](https://developer.nvidia.com/blog/cuda-10-features-revealed/) to remove CPU overhead associated with
-launching CUDA kernels sequentially. To enable the usage of CUDA Graphs, use the provider option as shown in the samples
-below.
+launching CUDA kernels sequentially. To enable the usage of CUDA Graphs, use the provider options as shown in the samples
+below. ORT supports multi-graph capture capability by passing the cuda graph annotation id to the run options. If the 
+cuda graph annotation id value is set to -1, cuda graph capture/replay is disabled in that run. User are not expected to 
+set the value to 0 as it is reserved for internal use.
```

Comment:
Add a line about currently not being able to delete captured cuda graphs so that it is expliclty known to users ?